Package.swift

```diff
@@ -113,19 +113,25 @@ let package = Package(
         ),
     ],
     targets: [
-        // The `PackageDescription` targets define the API which is available to
-        // the `Package.swift` manifest files. We build the latest API version
-        // here which is used when building and running swiftpm without the
-        // bootstrap script.
+        // The `PackageDescription` target provides the API that is available
+        // to `Package.swift` manifests. Here we build a debug version of the
+        // library; the bootstrap scripts build the deployable version.
         .target(
-            /** Package Definition API */
             name: "PackageDescription",
             swiftSettings: [
-                .define("PACKAGE_DESCRIPTION_4_2"),
+                .unsafeFlags(["-package-description-version", "5.5"]),
+                .unsafeFlags(["-enable-library-evolution"])
```

Comment:
It’s possible that using unsafe flags here is problematic for local development. What we really need to set here is package-description-version 5.5, because otherwise the implementation isn’t allowed to access its own properties that were unavailable in 4.0.  If there’s a better way to do this, that would be great.

Reply:
does it need to be 5.5. or latest?

Reply:
That's a good point — in main this should perhaps be `999`, if that's allowed on the command line.  I'll check.

Reply:
or if there is a constant we can use that is exposed to the API